padd.sh

```diff
@@ -588,6 +623,9 @@ PrintLogo() {
   elif [ "$1" = "mini" ]; then
     CleanEcho "${padd_text}${dim_text}mini${reset_text}  ${mini_status_}"
     CleanEcho ""
+  elif [ "$1" = "tiny" ]; then
+    CleanEcho "${bold_text} ${reset_text}${padd_text}${dim_text}tiny ${reset_text}Pi-hole® ${core_version_heatmap}v${core_version}${reset_text}, Web ${web_version_heatmap}v${web_version}${reset_text}, FTL ${ftl_version_heatmap}v${ftl_version}${reset_text}"
+    CleanPrintf "          PADD ${padd_version_heatmap}${padd_version}${reset_text} ${tiny_status_}${reset_text}\e[0K\\n"    
```

Comment:
If you add a space before `Pi-hole®` and a space before `PADD` you will get a more aesthetically pleasing alignment: 
![zz](https://user-images.githubusercontent.com/1605754/91642210-17902680-e9ef-11ea-8d42-5a17775132e6.png)

Thanks for your patch btw!

Reply:
Let me know if you want to add this in as a change, I'm good to merge this PR in and get things out. I haven't checked it fully yet, but I think we can get it deployed and make changes if needed later.

Reply:
@christianboyle I can merge this PR as it is and get the existing code out there. Would you open a PR with the improvements after this is on the master branch?

Reply:
@dschaper Sure, that sounds good.